WordPress/src/test/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelperTest.kt

```diff
@@ -179,6 +180,36 @@ class PostListItemUiStateHelperTest {
                 post = createPostModel(isLocallyChanged = true, status = POST_STATE_PRIVATE),
                 uploadStatus = createUploadStatus(uploadError = UploadError(MediaError(AUTHORIZATION_REQUIRED)))
         )
+        assertThat(state.data.statuses).contains(UiStringRes(R.string.error_media_recover_post_not_published))
```

Comment:
TIL there's also a `containsOnly` matcher so we can avoid the `contains` + `hasSize(1)` dance. 🎉 

```kt
        assertThat(state.data.statuses).containsOnly(UiStringRes(R.string.error_media_recover_post))
```

Reply:
Thanks! updated in b27868a